include/sdbus-c++/IObject.h

```diff
@@ -61,6 +61,11 @@ namespace sdbus {
     public:
         virtual ~IObject() = default;
 
+        /*!
+         * @brief return object path of the underlying DBus object
+         */
+        virtual const std::string& getObjectPath() const = 0;
```

Comment:
Would you please move the declaration of the function to the bottom of the class? This also has the advantage that the binary interface will remain backwards compatible with this change.

Reply:
Done